src/test/java/com/meilisearch/integration/SettingsTest.java

```diff
@@ -125,6 +123,50 @@ public void testUpdateSettingsTypoTolerance() throws Exception {
         assertTrue(newSettings.getTypoTolerance().isEnabled());
     }
 
+    @Test
+    @DisplayName("Test update multiple settings in a row")
+    public void testUpdateMultipleSettingsInARow() throws Exception {
+        Index index = createIndex("updateMultipleSettingsInARow");
+        Settings settingsDisplayedAttr = new Settings();
+        settingsDisplayedAttr.setDisplayedAttributes(
+                new String[] {"title", "overview", "genres", "release_date"});
+        index.waitForTask(index.updateSettings(settingsDisplayedAttr).getUid());
+        Settings newSettingsDisplayedAttr = index.getSettings();
+
+        Settings settingsRankingRules = new Settings();
+        settingsRankingRules.setRankingRules(
+                new String[] {
+                    "typo",
```

Comment:
Can you just have one or two? Because I'm not sure if the Meilisearch defaults are the same you choose here to test.

Reply:
`Meilisearch` has 6 by default:
```
        assertEquals(4, newSettingsDisplayedAttr.getDisplayedAttributes().length);
        assertEquals(6, newSettingsDisplayedAttr.getRankingRules().length);
        assertEquals(0, newSettingsDisplayedAttr.getSynonyms().size());
        assertEquals(4, newSettingsRankingRules.getDisplayedAttributes().length);
        assertEquals(8, newSettingsRankingRules.getRankingRules().length);
        assertEquals(0, newSettingsRankingRules.getSynonyms().size());
        assertEquals(4, newSettingsSynonyms.getDisplayedAttributes().length);
        assertEquals(2, newSettingsSynonyms.getSynonyms().size());
        assertEquals(8, newSettingsSynonyms.getRankingRules().length);
```
That why we check 6 then 8